python/ray/experimental/state.py

```diff
@@ -352,69 +358,105 @@ def log_files(self):
 
         return ip_filename_file
 
-    def task_profiles(self, start=None, end=None, num=None):
+    def task_profiles(self, start=None, end=None, num_slice=None, fwd=True):
```

Comment:
Can you explain what the `fwd` argument is/does? Also document it in the doc string.